src/core/async_graphql_hyper.rs

```diff
@@ -21,6 +25,10 @@ pub trait GraphQLRequestLike: Hash + Send {
     where
         E: Executor;
 
+    async fn execute_jit(
+        self,
+        app_ctx: Arc<AppContext>,
+    ) -> crate::core::jit::Response<Value, Error>;
```

Comment:
We don't need this method nor should this file be touched. Should implement `Executor` for a new `JITExecutor` and pass it as an argument here.